apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -924,14 +924,28 @@ public partial class BigtableClientImpl
     {
         private const string ResourcePrefixHeader = "google-cloud-resource-prefix";
 
+        private CallSettings _idempotentMutateRowSettings;
+
+        partial void OnConstruction(Bigtable.BigtableClient grpcClient, BigtableSettings effectiveSettings, ClientHelper clientHelper)
+        {
+            _idempotentMutateRowSettings = CallSettings.FromCallTiming(
+                CallTiming.FromRetry(effectiveSettings.IdempotentMutateRowRetrySettings));
+        }
+
         partial void Modify_ReadRowsRequest(ref ReadRowsRequest request, ref CallSettings settings) =>
             ApplyResourcePrefixHeader(ref settings, request.TableName);
 
         partial void Modify_SampleRowKeysRequest(ref SampleRowKeysRequest request, ref CallSettings settings) =>
             ApplyResourcePrefixHeader(ref settings, request.TableName);
 
-        partial void Modify_MutateRowRequest(ref MutateRowRequest request, ref CallSettings settings) =>
+        partial void Modify_MutateRowRequest(ref MutateRowRequest request, ref CallSettings settings)
+        {
+            if (request.IsIdempotent())
```

Comment:
If the caller has passed in other timing settings already, I'm not sure we'd want to change them. I suspect you can keep `_idempotentMutateRowSettings` as a `CallTiming`, and then use:

```csharp
if (request.IsIdempotent() && settings?.Timing == null)
{
    settings = settings.WithCallTiming(_idempotentMutateRowSettings);
}
```

Does that sound right to you?

Reply:
I was under the impression that MergedWith would only take the receiver's settings if they were not set in the argument's. The comments say "with overlaid taking priority". If the caller passed in other timing settings, won't those be given priority?

Reply:
Ah, of course, you're right. Sorry!